store/tikv/tikv_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tikv
+
+import (
+	"go/build"
+	"os"
+	"path"
+	"reflect"
+	"strings"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/parser"
+)
+
+type testTiKVSuite struct {
+}
+
+var _ = Suite(&testTiKVSuite{})
```

Comment:
I'm not sure this test really runs, no `TestT` and no `SetUpSuite`, please check it.

Reply:
Confirmed, it runs:

1. checked out 1 commit before PR #5834 
2. git cherry-pick this commit
3. `make test`:

```
...
FAIL: tikv_test.go:51: testTiKVSuite.TestParserNoDep

tikv_test.go:64:
    c.Assert(importName == parserPkgName, IsFalse,
        Commentf("`%s` is imported from `%s`, which is a child dependency of `%s`", parserPkgName, pkgName, currentPkgName))
... obtained bool = true
... `github.com/pingcap/tidb/parser` is imported from `github.com/pingcap/tidb/table/tables`, which is a child dependency of `github.com/pingcap/tidb/store/tikv`
...
```